tests/sparseml/pytorch/utils/test_verbosity_helper.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
We'll need to additionally add an integration test within the modifiers to make sure that the logging is doing what we intend